airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1252,7 +1263,10 @@ class BigQueryCreateEmptyTableOperator(BaseOperator):
         If set as a sequence, the identities from the list must grant
         Service Account Token Creator IAM role to the directly preceding identity, with first
         account from the list granting this role to the originating account (templated).
-    :param exists_ok: If ``True``, ignore "already exists" errors when creating the table.
+    :param exists_ok: Deprecated - use `if_exists="ignore"` instead.
+    :param if_exists: What should Airflow do if the table exists. If set to `log`, the TI will be passed to
+        success and an error message will be logged. Set to `ignore` to ignore the error, set to `fail` to
+        fail the TI, and set to `skip` to skip it.
```

Comment:
Small nit, I think it is a good idea move deprecated parameters into the bottom of arguments lists and because operator only supports keyword arguments, this is not break any pipeline.

Reply:
It's a good idea, I did it for all deprecated parameters in the file and not just in the operators I changed.